packages/storage/src/indexed-db/index.ts

```diff
@@ -477,6 +477,9 @@ export class IndexedDb implements IndexedDbInterface {
     const previousEpoch = cursor?.value ? Epoch.fromJson(cursor.value) : undefined;
     const index = previousEpoch?.index !== undefined ? previousEpoch.index + 1n : 0n;
 
+    // avoid saving the same epoch twice
+    if (previousEpoch?.startHeight === startHeight) return;
+
```

Comment:
At first I thought it would be best to link to `epochRoot`. 
But it seems that binding to a unique `startHeight` is just as good, plus I'm not sure that `epochRoot` is guaranteed to be unique for every epoch